lib/code-examples.js

```diff
@@ -1,109 +1,124 @@
 export default {
   'basic-interval': {
     name: 'Basic interval',
-    code: `Rx.Observable
-  .interval(1000)
-  .take(4)
+    code: `const { interval } = Rx.sources;
+const { take } = Rx.operators;
+
+interval(1000).pipe(take(4));
 `,
     timeWindow: 5000
   },
   'random-error': {
     name: 'Random error',
-    code: `Rx.Observable
-  .create(observer => {
-    let n = 1;
-
-    const intervalId = setInterval(() => {
-      if (Math.random() < 0.8 && n < 9) {
-        observer.next(n * n);
-        n += 1;
-      } else {
-        observer.error('Oh no...');
-      }
-    }, 1000);
-
-    return () => clearInterval(intervalId);
+    code: `const { Observable } = Rx.sources;
+const { take } = Rx.operators;
+Observable.create(observer => {
+  let n = 1;
+  const intervalId = setInterval(() => {
+    if (Math.random() < 0.8 && n < 9) {
+      observer.next(n * n);
+      n += 1;
+    } else {
+      observer.error('Oh no...');
+    }
+  }, 1000);
+
+  return () => clearInterval(intervalId);
   })
 `,
     timeWindow: 10000
   },
   'chess-game': {
     name: 'Chess game',
-    code: `const timer$ = Rx.Observable.interval(1000);
-const pieces$ = Rx.Observable.of('', '♞', '', '♞', '♘', '♞');
-const columns$ = Rx.Observable.of('e', 'c', 'g', 'd', 'e', 'f');
-const rows$ = Rx.Observable.of('4', '6', '4', '4', '2', '3');
-
-Rx.Observable.zip(
-  timer$,
-  pieces$,
-  columns$,
-  rows$,
-  (_, piece, column, row) => \`\${piece}\${column}\${row}\`
-)
+    code: `const { of, interval } = Rx.sources;
+const { zip, map } = Rx.operators;
+const timer$ = interval(1000);
+const pieces$ = of('', '♞', '', '♞', '♘', '♞');
+const columns$ = of('e', 'c', 'g', 'd', 'e', 'f');
+const rows$ = of('4', '6', '4', '4', '2', '3');
+timer$.pipe(
+  zip(
+    pieces$,
+    columns$,
+    rows$
+  ),
+  map(([timer, piece, column, row]) => {
+    return \`\${piece}\${column}\${row}\`
+  })
+);
 `,
     timeWindow: 7000
   },
   'higher-order-observable': {
     name: 'Higher order Observable',
-    code: `Rx.Observable
-  .interval(1000)
-  .groupBy(n => n % 2)
+    code: `const { interval } = Rx.sources;
+const { groupBy } = Rx.operators;
+
+interval(1000).pipe(groupBy(n => n % 2));
 `,
     timeWindow: 10000
   },
   'grouped-fibonacci': {
     name: 'Grouped Fibonacci',
-    code: `Rx.Observable
-  .interval(1000)
-  .scan(({ secondLast, last }) => ({
-    secondLast: last,
-    last: last + secondLast,
-  }), { secondLast: 0, last: 1 })
-  .pluck('secondLast')
-  .groupBy(n => Math.floor(Math.log10(n)))
+    code: `
+const { interval } = Rx.sources;
+const { scan, pluck, groupBy } = Rx.operators;
+
+interval(1000).pipe(
+  scan(
+    ({ secondLast, last }) => ({
+      secondLast: last,
+      last: last + secondLast
+    }),
+    { secondLast: 0, last: 1 }
+  ),
+  pluck("secondLast"),
+  groupBy(n => Math.floor(Math.log10(n)))
+);
+    
 `,
     timeWindow: 15000
   },
   'today-is': {
```

Comment:
This example behaves quite differently now. Mind checking why?

Reply:
Thanks for catching that ! Forgot to commit my change after renaming a variable.